### DISCUSSION TITLE
How to handle response?

### DISCUSSION THREAD

**Discussion by Borgelin** [openai/openai-python#884] [Q&A] [answered]
I have an app in Anvil that creates some questions from text i send through the API, and with the previous library it worked well.
But now with the post v1 (1.3.5 to be exact) my method no longfer works and I have no idea how to fix this.
The prompt is:

```
completions = client.chat.completions.create(
        model=model,
        messages=[
            {"role": "system", "content": f"""Generate {num_questions} <REST OF PROMPT>"""}
        ],
        max_tokens=8196,
        n=1,
        stop=None,
        temperature=0.1,
    )

    questions = completions.choices[0].message['content'].strip().split("\n\n")
    anvil.server.task_state['result'] = questions
    
    return questions
```

I worked out the rest to resemble the new documentation and it works fine until I reach this line:
`questions = completions.choices[0].message['content'].strip().split("\n\n")`

I get the error:
**TypeError: 'ChatCompletionMessage' object is not subscriptable**

How do I handle the "new" response?

**Comment by rattrayalex** [comment, marked as answer]
You want `completions.choices[0].message.content.strip().split("\n\n")`
